WebCryptoAPI/import_export/importKey_failures.js

```diff
@@ -267,4 +267,23 @@ function run_test(algorithmNames) {
             });
         });
     });
+
+    // Use an 'alg' field with incorrect casing.
+    testVectors.forEach(function(vector) {
+        var name = vector.name;
+        if (name === "X25519")
+            return; // "X25519" ignores the 'alg' field.
```

Comment:
The same is true for X448. And, perhaps it's clearer to write this the other way around, e.g. `if (name === "Ed25519" || name === "Ed448") { // ... do the below`?

Also because, for ECDH I'd expect the below to introduce a new failure as well since `"ECDH" === "ECDH".toUpperCase()`... but probably I'm misunderstanding something about how this code is called :sweat_smile:

Reply:
Or alternatively, if you prefer you could do:
```suggestion
        if (name !== "Ed25519" && name !== "Ed448")
            return;
```

Reply:
> Also because, for ECDH I'd expect the below to introduce a new failure as well since `"ECDH" === "ECDH".toUpperCase()`... but probably I'm misunderstanding something about how this code is called 😅

I'm afraid with this sentence we have discovered a bug with this test. For the "jwk" test cases we define an object called "data" with the same properties used in the JWK format. The 'd' property is duplicated there, while the 'x' one is missing. 

This error doesn't affect OKP test cases, but causes all the EC test cases to fail with DataError, no matter the test case. 

I think it'd be better to fix that bug in a separate PR before continue reviewing this.

Reply:
Yes, and Ed448 :)